src/platforms/ruby/common/configuration/options.mdx

```diff
@@ -216,7 +220,7 @@ config.trusted_proxies = ["2.2.2.2"]
 
 : You can gain more control on tracing event (transaction)'s sampling decision by providing a callable object (`Proc` or `Lambda`) as a `traces_sampler`:
 
-<PlatformContent includePath="performance/dynamic-sampling-function-intro" />
+<PlatformContent includePath="performance/traces-sampler-as-sampler" />
```

Comment:
👍  Good fix, the page was renamed to `src/includes/performance/sampling-function-intro/ruby.mdx` in 3e3980e907fc62d707d408c8c193c2e6cd183fb2 and so this got broken. Surprised the Gatsby build and CI has not caught this bad reference.